src/backend/marsha/core/lti.py

```diff
@@ -28,30 +28,36 @@ def __init__(self, request):
 
         """
         self.request = request
-        self._is_verified = False
 
     def verify(self):
         """Verify the LTI request.
 
         Raises
         ------
         LTIException
-            Exception raised if request validation fails
+            Raised if request validation fails
+        ImproperlyConfigured
+            Raised if BYPASS_LTI_VERIFICATION is True but we are not in DEBUG mode
 
         Returns
         -------
-        boolean
-            True if the request is a valid LTI launch request
-            or if BYPASS_LTI_REQUEST_SIGNATURE is True
+        string
+            It returns the consumer site name related to the passport used in the LTI launch
+            request if it is valid.
+            If the BYPASS_LTI_VERIFICATION and DEBUG settings are True, it returns "example.com"
+            as the consumer site to which all test videos will be linked.
 
         """
-        if BYPASS_LTI_REQUEST_SIGNATURE:
-            return True
+        if settings.BYPASS_LTI_VERIFICATION:
+            if not settings.DEBUG:
+                raise ImproperlyConfigured(
+                    "Bypassing LTI verification only works in DEBUG mode."
+                )
+            return ConsumerSite.objects.get_or_create(name="example.com")[0]
```

Comment:
Or maybe we could take the consumer site from the `resource_link_id` passed in the POST request? This gives us more flexibility to simulate situations during development

Reply:
Good idea ! Let me do it then you can squash @lunika